fw/procfs.h

```diff
@@ -109,4 +116,27 @@ DECLARE_PER_CPU_ALIGNED(TfwPerfStat, tfw_perfstat);
 #define TFW_ADD_STAT_BH(val, ...)	\
 		this_cpu_add(tfw_perfstat.__VA_ARGS__, val)
 
+static inline void
+tfw_inc_global_hm_stats(int status)
+{
+	int i;
+	TfwPerfStat *this_stat = this_cpu_ptr(&tfw_perfstat);
+	TfwHMStats *hm = this_stat->hm;
+
+	if (!hm)
```

Comment:
In what case we can have `hm == NULL` here?

Reply:
When `health_stat` option is not set